src/oasis-indexer/api.ts

```diff
@@ -36,7 +36,36 @@ export const useGetRuntimeTransactions: typeof generated.useGetRuntimeTransactio
         (data: generated.RuntimeTransactionList) => {
           return {
             ...data,
-            transactions: (data.transactions || []).map(tx => {
+            transactions: data.transactions.map(tx => {
+              return {
+                ...tx,
+                fee: tx.fee ? fromBaseUnits(tx.fee, paraTimesConfig.emerald!.decimals) : undefined,
```

Comment:
We have a parameter called `runtime`, but we are still using hard-coded emerald values here. Is that intentional?

Reply:
Yeah, I'll leave that to sapphire migration PRs